src/components/ebay-tooltip/test/test.browser.js

```diff
@@ -26,7 +26,7 @@ describe('given the default tooltip', () => {
             expect(component.emitted('expand')).has.length(1);
         });
 
-        describe('when the host element loses hover', () => {
+        describe.skip('when the host element loses hover', () => {
```

Comment:
Why is this test skipped?

Reply:
It kept failing randomly. One run through it would pass, and then another it would fail.

Reply:
I think we should leave it as is, and then revisit it if it's failing in another PR. 
If we keep seeing it fail and retest it's fine since we will want to fix it, but removing it will make us lose coverage and we might forget about it

Reply:
okay I unskipped it 👍